language/e2e-tests/Cargo.toml

```diff
@@ -21,8 +21,9 @@ libra-state-view = { path = "../../storage/state-view", version = "0.1.0" }
 libra-types = { path = "../../types", version = "0.1.0", features = ["fuzzing"] }
 libra-workspace-hack = { path = "../../common/workspace-hack", version = "0.1.0" }
 move-core-types = { path = "../move-core/types", version = "0.1.0" }
-move-vm-types = { path = "../move-vm/types", version = "0.1.0", features = ["debug_module"] }
+move-vm-natives = { path = "../move-vm/natives", version = "0.1.0", features = ["debug_module"] }
 move-vm-runtime = { path = "../move-vm/runtime", version = "0.1.0", features = ["debug_module"] }
+move-vm-types = { path = "../move-vm/types", version = "0.1.0", features = ["debug_module"] }
```

Comment:
not sure we need to pass this to anybody except natives but it does not seem to create issues so we may leave it as is

Reply:
If I remember correctly feature flags don't get propagated to dependencies so I think you can remove `features = ["debug_module"]` for move-vm-types. (Also remember to delete it from the [feature] section in cargo.toml.)